models/actions/runner.go

```diff
@@ -52,14 +52,50 @@ type ActionRunner struct {
 	Deleted timeutil.TimeStamp `xorm:"deleted"`
 }
 
-func (r *ActionRunner) OwnType() string {
+// RunnerType defines the runner type
+type RunnerType int
+
+const (
+	// RunnerTypeGlobal defines a global runner
+	RunnerTypeGlobal RunnerType = iota
+
+	// RunnerTypeOrganization defines an organization runner
+	RunnerTypeOrganization
+
+	// RunnerTypeRepository defines a repository runner
+	RunnerTypeRepository
+)
+
+func (r *ActionRunner) Type() RunnerType {
+	if r.RepoID != 0 {
+		return RunnerTypeRepository
+	}
+	if r.OwnerID != 0 {
+		return RunnerTypeOrganization
+	}
+	return RunnerTypeGlobal
+}
+
+func (rt RunnerType) String() string {
+	switch rt {
+	case RunnerTypeGlobal:
+		return "Global"
+	case RunnerTypeOrganization:
+		return "Organization"
+	case RunnerTypeRepository:
+		return "Repository"
+	}
+	return fmt.Sprintf("Unknown Type %d", rt)
+}
```

Comment:
Hmm…
Yes and no.
This works. At least for the English language.
However, we can also think about whether we want to run the result through `ctx.Tr`.
Or does an English-only approach suffice here?

Reply:
Should be localized

Reply:
Done.